crates/telio-traversal/src/endpoint_providers/upnp.rs

```diff
@@ -515,13 +514,12 @@ impl<Wg: WireGuard, I: UpnpEpCommands, E: Backoff> State<Wg, I, E> {
                 }
             }
             Err(e) => {
-                if let Error::UpnpError(HttpErrorCode(http::StatusCode::INTERNAL_SERVER_ERROR)) = e
-                {
+                if let Error::NoIGDGateway = e {
```

Comment:
Why was this `if` section not removed, but instead a different error is checked here?

Reply:
I thought that this is the error which should be considered here, as this is the case in which there is no point in deleting routes when there are none - this delete)endpoint_routes would fail anyway